src/Eccube/Controller/ShoppingController.php

```diff
@@ -1410,7 +1422,20 @@ private function customerValidation(Application $app, array $data)
         $errors[] = $app['validator']->validateValue($data['customer_name02'], array(
             new Assert\NotBlank(),
             new Assert\Length(array('max' => $app['config']['name_len'],)),
-            new Assert\Regex(array('pattern' => '/^[^\s ]+$/u', 'message' => 'form.type.name.firstname.nothasspace'))
+            new Assert\Regex(array('pattern' => '/^[^\s ]+$/u', 'message' => 'form.type.name.lastname.nothasspace'))
+        ));
+        $data['customer_kana01'] = Converter::convertToKana($data['customer_kana01']);
```

Comment:
I think this soursecode is OK.

before fixing: `Converter::convertToKana($data['customer_kana01']);`
my solution: `mb_convert_kana($data['customer_kana01'], 'CV', 'utf-8');`